frontend/app/components/TaskDescription.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { FetchError } from 'ofetch'
+
+const props = defineProps<{
+  taskId: string | string[] | undefined
+}>()
+
+const description = ref('')
+const toast = useToast()
+
+try {
+  const task_id = String(props.taskId)
+  const address = '/tasks/description/{task_id}'
+  const { data: response } = await useApi(address, {
+    path: { task_id },
+    key: `task-description-${task_id}`,
+  })
+
+  if (response.value === undefined) {
+    showError({
+      statusCode: 404,
+      message: 'Zadanie nie zostało znalezione',
+    })
+    console.error('Task not found')
+  } else {
+    description.value = String(response.value)
+  }
+} catch (error) {
+  console.error(error)
+  if (!(error instanceof FetchError)) {
```

Comment:
I think it would be better to redirect to error page than showing blank page with a toast